service.subtitles.legendasdivx/service.py

```diff
@@ -448,10 +478,17 @@ def Download(id, filename):
             xbmc.sleep(500)
         except: log(u"Failed to save subtitles to '%s'" % (local_tmp_file,))
         if packed:
-            xbmc.executebuiltin("XBMC.Extract(%s, %s)" % (local_tmp_file.encode("utf-8"), _newtemp))
-            xbmc.sleep(1000)
-
+            try:
+                compressed_file = 'rar://' + quote_plus(local_tmp_file) + '/'
+                log(u"Will try to extract...")
+                xbmc_extract(compressed_file,_newtemp)
+                log(u"Success...")
+            except:
+                xbmc.executebuiltin("XBMC.Extract(%s, %s)" % (local_tmp_file.encode("utf-8"), _newtemp))
```

Comment:
XBMC. prefixes were removed from builtins in Matrix. I hope the version in matrix doesn't include them, for this one it's fine

Reply:
Need to check that before next release on the matrix branch.

Reply:
Yeah that one slipped between our fingers :) I confirmed you need to change it in matrix